library/src/main/java/org/xmtp/android/library/Dm.kt

```diff
@@ -41,7 +41,7 @@ class Dm(
         get() = Date(libXMTPGroup.createdAtNs() / 1_000_000)
 
     val peerInboxId: InboxId
-        get() = libXMTPGroup.dmPeerInboxId()
+        get() = libXMTPGroup.dmPeerInboxId() ?: ""
```

Comment:
@cameronvoell this is nullable now. What do you think would be the best solution? It would be a breaking change to make it `InboxId?` What case would it be empty?